xml/Microsoft.Extensions.Logging.AzureAppServices/BatchingLoggerProvider.xml

```diff
@@ -25,7 +25,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>A provider of <see cref="T:Microsoft.Extensions.Logging.AzureAppServices.BatchingLogger" /> instances.</summary>
+    <summary>A provider of <see cref="T:Microsoft.Extensions.Logging.AzureAppServices.Internal.BatchingLogger" /> instances.</summary>
```

Comment:
Wondering if comment https://github.com/dotnet/dotnet-api-docs/pull/4922#issuecomment-708646224 is also relevant here.

Reply:
Correct. This is the fix based on that comment.